KS.Fiks.IO.Arkiv.Client/Schema/sok.xsd

```diff
@@ -139,6 +139,9 @@
             <xs:enumeration value="sak.planident.fylke.fylkesnummer"/>
             <xs:enumeration value="sak.planident.kommune.kommunenummer"/>
             <xs:enumeration value="sak.planident.planidentifikasjon"/>
+            
+            <xs:enumeration value="sak.part.organisasjonsid"/>
+            <xs:enumeration value="sak.part.personid"/>
```

Comment:
Det kan hende at navnet må endres hvis vi gjør om til et objekt for personid for type og id. Ser ingen grunn til å komplisere som i GI med mulighet for flere ID-er. Spørsmålet er om det skal være felt for type da eller om personid skal kunne ha forskjellige typer (fødselsnummer, D-nummer, ...).

Reply:
@sturtzel Jeg tror vi kan la denne stå som dette med to forskjellige typer da det ble slik i `arkivmelding.xsd`? 
https://github.com/ks-no/fiks-arkiv-client-dotnet/pull/28/files#diff-fee360476aff5034e3619b5f4a7c53abd477415c790b7f1d9444d4ca7406b3da

Reply:
Ja, basert på det vi ble enige om på torsdag er dette OK.